nixos/modules/misc/documentation.nix

```diff
@@ -176,7 +200,7 @@ in
       enable = mkOption {
         type = types.bool;
         default = true;
-        description = lib.mdDoc ''
+        description = mdDoc ''
```

Comment:
@pennae @infinisil; I guess these `mdDoc` markers are not needed anymore?
Since @philiptaron is cleaning the `with` statements we could also remove this unneded filewide "global"